Data/ODBC/testsuite/src/ODBCSQLServerTest.cpp

```diff
@@ -59,7 +59,7 @@ using Poco::DateTime;
 	#pragma message ("Using " MS_SQL_SERVER_ODBC_DRIVER " driver")
 #else
 	#define MS_SQL_SERVER_ODBC_DRIVER "FreeTDS"
-	#define FREE_TDS_VERSION "7.4"
+	#define FREE_TDS_VERSION "8.0"
```

Comment:
there is no FreeTDS 8.0
https://github.com/pocoproject/poco/blob/3da8ee633e0e028d530f282fe4ab6869de1e15b2/Data/ODBC/testsuite/src/ODBCSQLServerTest.cpp#L50
In any case, FreeTDS is legacy, not necessary at all since MS SQL Server ODBC driver now works on Linux, we don't need FreeTDS anymore (it can stay for anyone who may be using it, but default should be MS SQL Server ODBC driver 18 and FreeTDS version should be reverted back to 7.4)